README.md

```diff
@@ -51,7 +51,7 @@ steps:
 
 | name | description | required |
 | ----------- | ----------- | ----------- |
-| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. | false |
+| `version-file` | Specify a path to search for the unity project version text file. Useful if there are multiple projects in a single repo. Pass `None` if creating a new project to skip file search. | false |
```

Comment:
how about changing this to just project-path, and deriving the version file path from there?

Reply:
that'd be a breaking change imho, and would rather keep it like it is to onboard people using the older versions of this action.

Reply:
I don't really think it matters too much, if you have the project path, you can get the version file, and if you have the version file you can get the project path.